packages/stex-react-renderer/src/lib/ProblemDisplay.tsx

```diff
@@ -406,27 +406,46 @@ function groupingByBloomDimension(data?: string) {
   return groupedData;
 }
 
-function DimAndURIListDisplay({ title, data }: { title: string; data?: string }) {
-  const groupedData = groupingByBloomDimension(data);
+function isBloomDimensionFormat(data: string) {
```

Comment:
You shouldn't need this function. You know the format of the data when it is created - whether it has bloom dimensions or not.